docs/en/guide/gateway.md

```diff
@@ -577,12 +577,12 @@ USAGE:
   mc admin group COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]
 
 COMMANDS:
-  add      add users to a new or existing group
-  remove   remove group or members from a group
-  info     display group info
-  list     display list of groups
-  enable   enable a group
-  disable  disable a group
+  add      Add users to a new or existing group
+  remove   Remove a group or its members
```

Comment:
@timfeirg PTAL

Reply:
this content is extracted directly from minio client output, do not modify @CaitinChen